src/transformScript.js

```diff
@@ -32,7 +31,8 @@ function transformShouldBeBool() {
           path.node.callee.name = transformBoolMap[path.node.callee.name];
 
           const newActual =
-            babel.template.expression(path.node.arguments[0].value)();
+            babel.template.expression(path.node.arguments[0].value,
+                {placeholderPattern: false})();
```

Comment:
We've discussed why this is needed offline, but it'd be good not to repeat this everywhere. Could you make a wrapper and add some comment for why this is needed there?

Tangent: I wonder if we can use https://babeljs.io/docs/en/babel-parser#babelparserparseexpressioncode-options instead.

Reply:
Importing `@babel/parser` and using parseExpression worked, and I think that's better. Replaced everywhere. Thanks for finding this!